docs/developer-docs/multi-chain/chain-key-tokens/ckerc20/making-transactions.mdx

```diff
@@ -0,0 +1,126 @@
+
+:::
+
+If the targeted ckERC20 token is listed, the first step is to create an Ethereum transaction that calls the `approve` function on the corresponding ERC-20 contract address, allowing the helper smart contract to withdraw some of the user's funds.
+
+The next step is to call the `deposit` function of the helper smart contract, specifying the ERC-20 smart contract address, the amount to be deposited, and the principal ID that should be credited for the deposit. The principal ID must be encoded as a `bytes32` array. A convenient way to obtain the correct encoding is to use the `Principal → Bytes32 conversion` function on the ckETH minter [dashboard](https://sv3dd-oaaaa-aaaar-qacoa-cai.raw.icp0.io/dashboard).
+
+:::warning
+
+If your encoded principal ID is incorrect, funds will be lost.
+
+:::
+
+No further user action is required. The `icrc1_balance_of` function can be called on the corresponding ckERC20 ledger to check if the ckERC20 tokens have been minted, which usually takes about 20 minutes.
+
+
+## Converting ckERC20 tokens to ERC-20 tokens
+
+Since the user must pay the transaction fees, the `icrc2_approve` function must be called on the ckETH ledger, allowing the ckETH minter to spend some of the user's ckETH tokens.
+For example, the following command approves the ckETH minter to take up to 1 ckETH from the user's account.
+
+```
+dfx canister --ic call ledger icrc2_approve "(record { spender = record { owner = principal \"$(dfx canister id minter --ic)\" }; amount = 1_000_000_000_000_000_000:nat })"
+```
+
+The user must further approve the ckETH minter to transfer the desired amount of the targeted ckERC20 token:
+
+```
+dfx canister --ic call CKERC20_LEDGER icrc2_approve "(record { spender = record { owner = principal \"$(dfx canister id minter --ic)\" }; amount = CKERC20_TOKEN_AMOUNT:nat })"
+```
+
+The placeholder `CKERC20_LEDGER` must be replaced by the canister ID of the corresponding ckERC20 ledger and `CKERC20_TOKEN_AMOUNT` must be replaced by the desired amount, denominated in the smallest unit of the token. For example, USDC and ckUSDC use 6 decimals, so 1 USDC must be written as `1_000_000`.
+
+The last step is to call the `withdraw_erc20` function on the ckETH minter as follows.
+
+```
+dfx canister --ic call minter withdraw_erc20 "(record { amount = CKERC20_TOKEN_AMOUNT:nat; ckerc20_ledger_id = principal \"CKERC20_LEDGER\"; recipient = \"ETH_ADDRESS\"})"
+```
+
+As before, `CKERC20_TOKEN_AMOUNT` and `CKERC20_LEDGER` must be replaced by the desired amount in the smallest unit and the canister ID of the ckERC20 ledger, respectively.
+Lastly, `ETH_ADDRESS` must be replaced by the desired Ethereum destination address.
+
+## Transferring ckERC20 tokens
+
+ckERC20 tokens are standard ICRC-1 tokens and can be transferred by calling the `icrc1_transfer` function.
```

Comment:
>standard ICRC-1 tokens

The other pages refer to ckERC20 as an ICRC-2 compliant token, but then here reference it as a 'standard ICRC-1 token'. This is confusing, is there a reason why here it is specified as an ICRC-1 token and not ICRC-2? Could this be reworded to say that it complies with ICRC-1 and ICRC-2, and therefore can be transferred with the `icrc1_transfer` function?

Reply:
Good point. Done!